source/aggregation-tutorials/group-total.txt

```diff
@@ -0,0 +1,225 @@
+.. _node-aggregation-group-total:
+
+===============
+Group and Total
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: code example, node.js, analyze, aggregation
+
+Introduction
+------------
+
+In this tutorial, you can learn how to use the {+driver-short+} to
+group documents and create totals from these groups. The aggregation
+groups documents on a common field and combines information
+from the documents to create new fields.
```

Comment:
Suggestion:

I think "to" ("in order to") is followed by a purpose/objective, but in this case, it seems like more of a result (create new fields).

Perhaps formatting these items could make it easier to identify the separate actions, e.g.:


This aggregation performs the following operations:

- Filters documents by a field value
- Groups documents by a common field value
- Adds computed fields to each result